sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobOutputStreamTest.groovy

```diff
@@ -0,0 +1,74 @@
+package com.azure.storage.blob
+
+import com.azure.storage.common.Constants
+import spock.lang.Ignore
+
+class BlobOutputStreamTest extends APISpec {
+    private static int FOUR_MB = 4 * Constants.MB
+
+    @Ignore
```

Comment:
When do you want them run?
Could you make them run in live or leave the reason of ignore.

Reply:
These were always ignored, when they were in a JUnit test class the `@Test` was commented out.

Will look to enable these in a follow-up PR.